src/views/Viewer.vue

```diff
@@ -219,9 +225,10 @@ export default {
 					modal: this.components[mime],
 					loaded: false
 				}
+				this.updatePreviousNext()
+			} else {
+				console.error(`The following file could not be displayed because to view matches its mime type`, fileName, fileInfo)
```

Comment:
I feel this could be easier written?

Reply:
This is meant to be a dev debug. Since no regular user would open the console I guess :thinking: 
But I've always been terrible at writing error messages, so feel free to suggest a simpler one :grin:

Reply:
@skjnldsv Hey no worries. I thought it was just unclear what it said. I am actually struggling understanding: "because to view matches". What about this: "The following file could not be displayed due to it's mime type"?